src/Documentation/tutorials_and_samples/tutorial_1_helloworld.md

```diff
@@ -0,0 +1,377 @@
+---
+layout: page
+title: Tutorial 1 Hello World
+---
+
+# Tutorial 1: Hello World
+
+This process recreates the same Hello World sample application available [here](https://github.com/dotnet/orleans/tree/master/Samples/2.0/HelloWorld).
+
+## Overview of the parts
+
+This application consists of a solution that contains four projects: SiloHost, OrleansClient, HelloWorld.Interfaces, and HelloWorld.Grains.
```

Comment:
can we add sections doing overview for grain and grain interface concepts? This is the first tutorial our new user would see, it would be great if we can briefly touch grain and grain interface concepts, link it to its doc page. These two concepts are what our new users will be most interested in.

Reply:
Yes! I have paraphrased some of the info from the Grain and Grain Interface pages and linked them.